patches/fsharp/0002-Add-type-annotation-to-F.patch

```diff
@@ -0,0 +1,25 @@
+From 00de0de908a064acbb878802d4cb7ee1ced6ff43 Mon Sep 17 00:00:00 2001
+From: Chris Rummel <crummel@microsoft.com>
+Date: Thu, 7 Mar 2019 12:14:30 -0600
+Subject: [PATCH] Add type annotation to F#.
+
+---
+ src/utils/reshapedreflection.fs | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/src/utils/reshapedreflection.fs b/src/utils/reshapedreflection.fs
+index b1f0aeca8..33e9b0c73 100644
+--- a/src/utils/reshapedreflection.fs
++++ b/src/utils/reshapedreflection.fs
+@@ -360,7 +360,7 @@ module internal ReflectionAdapters =
+             this.GetType().GetMethod(methodName, argTypes, BindingFlags.Public)
+ 
+     type System.Char with
+-        static member GetUnicodeCategory(c) = System.Globalization.CharUnicodeInfo.GetUnicodeCategory(c)
++        static member GetUnicodeCategory(c) = System.Globalization.CharUnicodeInfo.GetUnicodeCategory(c : char)
```

Comment:
I'm curious why this is needed?  Is it a product issue?

Reply:
Yes, an unmodified F# at this commit does not build for me outside of source-build.  I think the buildfromsource projects there have suffered from some bitrot, I'm going to make a PR in F# to fix them.